scala.wake

```diff
@@ -12,13 +12,20 @@ def bloopWake = source "{here}/bloop_wake"
 # Where we expect to find wit/scalaplugin.py
 def scalaPluginPath = "wit/lib"
 
+global def myMakeStatePath x =
```

Comment:
I think this should not be `global`

Reply:
I agree. You should switch to using a 0.18 mechanism and packages, but for now this gets it to work.

Reply:
product-coreip-sifive uses it also.

Reply:
I would argue that each repo should have its own private implementation as the migration rather than adding an API that this repo is theoretically now responsible for

Reply:
fine.